files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -705,3 +705,14 @@
     block: |
       # Set default umask value.
       umask 027
+
+- file:
+    src: /bin/true
+    dest: /usr/lib/nullmailer/true
+    owner: foo
+    group: foo
```

Comment:
Should this be an actual user/group?

Reply:
shoot, I removed it, then editor crashed and didn't notice that.. let me remove it again :p